tests/Gedmo/SoftDeleteable/DatePointTest.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/*
+ * This file is part of the Doctrine Behavioral Extensions package.
+ * (c) Gediminas Morkevicius <gediminas.morkevicius@gmail.com> http://www.gediminasm.org
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Gedmo\Tests\SoftDeleteable;
+
+use Carbon\Doctrine\DateTimeType;
+use Doctrine\Common\EventManager;
+use Doctrine\DBAL\Types\Type as DoctrineType;
+use Doctrine\DBAL\Types\Types;
+use Gedmo\SoftDeleteable\Filter\SoftDeleteableFilter;
+use Gedmo\SoftDeleteable\SoftDeleteableListener;
+use Gedmo\Tests\SoftDeleteable\Fixture\Entity\BookDatePoint;
+use Gedmo\Tests\Tool\BaseTestCaseORM;
+use Symfony\Component\Clock\DatePoint;
+
+final class DatePointTest extends BaseTestCaseORM
```

Comment:
The tests in this class would need to be skipped if the date point type doesn't exist (i.e. someone runs the tests with Symfony 6.4 components loaded)